app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -213,14 +213,18 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['miqService', 'API
     // model attribute with AWS.
     vm.cloudVolumeModel.aws_volume_type = data.volume_type;
     vm.cloudVolumeModel.aws_availability_zone_id = data.availability_zone.ems_ref;
+    vm.cloudVolumeModel.aws_encryption = data.encrypted;
+    vm.cloudVolumeModel.aws_iops = data.iops;
 
     // If volume was created from snapshot and this snapshot still exists
     if (data.base_snapshot) {
       vm.cloudVolumeModel.aws_base_snapshot_id = data.base_snapshot.ems_ref;
     }
 
     // Update the IOPS based on the current volume size.
-    vm.sizeChanged(vm.cloudVolumeModel.size);
+    if (vm.cloudVolumeModel.aws_iops == undefined) {
```

Comment:
This would call ``sizeChanged`` when ``vm.cloudVolumeModel.aws_iops`` already has value other than undefined. And we need it for calculating undefined iops values.